packages/cli/src/templates/plugin/stencil.config.ts

```diff
@@ -0,0 +1,71 @@
+require("dotenv").config();
+
+import { Config } from "@stencil/core";
+import { reactOutputTarget } from "@stencil/react-output-target";
+import corejam from "@corejam/dev/dist/rollup";
+import fs from "fs";
+import replace from "@rollup/plugin-replace";
+
+const targets = process.env.targets?.split(",") || [];
+
+const config: Config = {
+  namespace: process.env.NODE_ENV === "production" ? "corejam-plugin-dershop" : "corejam-dev",
```

Comment:
here needs to be a plceholder for namespace, which cli is renaming based on name

Reply:
Whoops i actually had that in there but in a temp file. Shouldnt a require("package.json").name be ok too?

Reply:
jup sounds good